.pre-commit-config.yaml

```diff
@@ -18,7 +18,8 @@ repos:
                 - pep8-naming
                 - pydocstyle<4.0.0
     - repo: https://github.com/pre-commit/mirrors-mypy
-      rev: v0.701
+      rev: v0.770
       hooks:
           - id: mypy
-            exclude: tests/data/*
+            exclude: ^((tests|bin)\/)?[\w-]+\.py$
+            args: ["--disallow-untyped-defs", "--ignore-missing-imports"]
```

Comment:
Huh. The `--ignore-missing-imports` seems a little bit mysterious...

Reply:
It avoids errors imports where there are no hints/stubs:
```
sphinx_docs/conf.py:4: error: Cannot find implementation or library stub for module named 'tomlkit'
sphinx_docs/conf.py:96: error: Cannot find implementation or library stub for module named 'sphinx_rtd_theme'
sphinx_gherkindoc/sample-conf.py:39: error: Cannot find implementation or library stub for module named 'recommonmark.parser'
sphinx_gherkindoc/sample-conf.py:75: error: Cannot find implementation or library stub for module named 'sphinx_rtd_theme'
sphinx_gherkindoc/parsers/behave.py:4: error: Cannot find implementation or library stub for module named 'behave.parser'
sphinx_gherkindoc/parsers/behave.py:4: error: Cannot find implementation or library stub for module named 'behave'
sphinx_gherkindoc/parsers/pytest_bdd.py:6: error: Cannot find implementation or library stub for module named 'pytest_bdd.feature'
sphinx_gherkindoc/parsers/pytest_bdd.py:6: error: Cannot find implementation or library stub for module named 'pytest_bdd'
sphinx_gherkindoc/utils.py:6: error: Cannot find implementation or library stub for module named 'sphinx.util'
sphinx_gherkindoc/utils.py:6: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
sphinx_gherkindoc/utils.py:6: error: Cannot find implementation or library stub for module named 'sphinx'
sphinx_gherkindoc/writer.py:7: error: Cannot find implementation or library stub for module named 'behave.parser'
sphinx_gherkindoc/writer.py:7: error: Cannot find implementation or library stub for module named 'behave'
sphinx_gherkindoc/writer.py:8: error: Cannot find implementation or library stub for module named 'behave.model'
sphinx_gherkindoc/writer.py:9: error: Cannot find implementation or library stub for module named 'behave.model_core'
sphinx_gherkindoc/cli.py:10: error: Cannot find implementation or library stub for module named 'sphinx'
Found 15 errors in 7 files (checked 12 source files)
```

We can certainly go back and add stubs ourselves for all these, but it seems like enough scope to be its own work at some point (and work I'm not sure we honestly want to do).

Reply:
Maybe you should put a `files: "^sphink_gherkindoc/"` in the top of the file. That would ignore everything in the docs directory.

I think the `ignore-missing-imports` is pretty standard from what I've seen because so many packages do not have stubs (hasn't caught up with Typescript yet), but I'd prefer to see that in a `mypy.ini` file than as args here. I like `--disallow-untyped-defs` too.